src/pages-and-resources/discussions/data/redux.test.js

```diff
@@ -404,7 +404,7 @@ describe('Data layer integration tests', () => {
         plugin_configuration: {
           allow_anonymous: true,
           allow_anonymous_to_peers: true,
-          discussion_blackouts: [['2015-09-15', '2015-09-21'], ['2015-10-01', '2015-10-08']],
+          discussion_blackouts: [],
```

Comment:
everywhere in tests now we passing empty blackout dates array? is that because we want to execute test cases successfully? what if I want to mock data here?

Reply:
The test cases are not in this scope. Blackout dates mocking will take some time to seamline the things. So right now, I'm using empty [] to compete for the ticker scope. A proper mock date test case will add later.